packages/aws-s3/aws-s3.4.7.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+license: "BSD-3-Clause"
+homepage: "https://github.com/andersfugmann/aws-s3"
+dev-repo: "git+https://github.com/andersfugmann/aws-s3"
+bug-reports: "https://github.com/andersfugmann/aws-s3/issues"
+doc: "https://andersfugmann.github.io/aws-s3/"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.0.0"}
+  "ocaml-inifiles"
+  "digestif" {>= "0.7"}
+  "ptime"
+  "uri"
+  "ezxmlm" {>= "1.1.0"}
+  "ppx_protocol_conv_xmlm" {>= "5.0.0" & < "6.0.0"}
+  "ppx_protocol_conv_json" {>= "5.0.0" & < "6.0.0"}
```

Comment:
What is the reason behind these strict upper-bound constraints?
```suggestion
  "ppx_protocol_conv_xmlm" {>= "5.0.0"}
  "ppx_protocol_conv_json" {>= "5.0.0"}
```

Reply:
None other than a major version change will presumably have API breaking changes. But I can re-introduce the upper bound constraint  if/when that happend.